apps/crn-server/test/routes/research-outputs.route.test.ts

```diff
@@ -120,6 +138,124 @@ describe('/research-outputs/ route', () => {
         researchOutputId,
       );
     });
+    test('Should return 404 when you are trying to get a draft and are not part of the team', async () => {
+      userMockFactory.mockReturnValueOnce(createUserResponse());
```

Comment:
maybe be more specific here too as we don't really know which teams this user fixture contains? so yeah create user response but then modify the team list for the user to show it does not contain the team that the research output is assicated with